src/strategy/uniform.ts

```diff
@@ -169,17 +167,18 @@ export const applyUniformStrategy: ApplyStrategy<UniformStrategy> = (
           'uniform',
           graphDataType,
           ['code', 'data'],
-          true
+          true,
         ),
         (filler) => {
-          const mangledName = mangleName(name, graphNode, siblingNode);
+          //const mangledName = mangleName(name, graphNode, siblingNode);
+          const mangledName = name;
```

Comment:
This is the thing that kicked off this whole refactor - not requiring strategies to know about mangled names.